documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/indexing/Book.java

```diff
@@ -28,6 +28,8 @@
 	@IndexedEmbedded
 	private Author author;
 
+	private int year;
```

Comment:
`publicationYear` maybe? Otherwise it's not really clear what the year refers to.